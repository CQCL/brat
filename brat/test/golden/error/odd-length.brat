f(X :: *, n :: #, Vec(X, n)) -> Vec(X, n)
f(X, n, xs) = xs

g :: Vec(Nat, 4)
g = f(Nat, 4, [1] =, 2 ,= [3])
