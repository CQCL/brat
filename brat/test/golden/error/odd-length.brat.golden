Error in test/golden/error/odd-length.brat@FC {start = Pos {line = 5, col = 15}, end = Pos {line = 5, col = 30}}:
g = f(Nat, 4, [1] =, 2 ,= [3])
              ^^^^^^^^^^^^^^^

  Expected vector of length 4
from the type:  Vec(VApp VPar In checking_check_defs_1_g_1_ 0 B0, VPar In checking_check_defs_1_g_1_ 1)
but got vector: concatEqOdd([1], 2, [3])
of odd length


